Tella/Data/Database/Protocols/UwaziServerLanguageProtocol.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  UwaziServerLanguageProtocol.swift
+//  Tella
+//
+//  Created by Robert Shrestha on 9/11/23.
+//  Copyright © 2023 HORIZONTAL. All rights reserved.
+//
+
+import Foundation
+
+protocol UwaziServerLanguageProtocol {
+    func createLanguageTableForUwazi()
```

Comment:
In my opinion it's better to create a separate Uwazi Server table and it contains the locale cookie and connect cookie  instead to use tella web server table and add language table 
see Android implementation:    `createUZIServer` https://github.com/Horizontal-org/Tella-Android/blob/b941e38f9d62032e1471b6dfd5e467b44acca23d/mobile/src/main/java/rs/readahead/washington/mobile/data/database/UwaziDataSource.java#L319C80-L319C80